fivetran_provider/operators/fivetran.py

```diff
@@ -61,6 +63,7 @@ def __init__(
         fivetran_retry_delay: int = 1,
         connector_id: str = None,
         poll_frequency: int = 15,
+        manual=True,
```

Comment:
minor but should be more like:

manual: bool = True

We aren't running the type checking software so it doesn't matter right now.